test/NuGet.Clients.Tests/NuGet.CommandLine.Test/NuGetPackCommandTest.cs

```diff
@@ -3063,9 +3063,9 @@ public string F2(int a)
             }
         }
 
-        // Tests that pack works with -MSBuildVersion set to 12
-        [Fact]
-        public void PackCommand_WithMsBuild12()
+        // Tests that pack works with -MSBuildVersion set to 15.1
+        [Fact(Skip = "Re-enable this when MSBuild 15.1 is installed on CI machines")]
+        public void PackCommand_WithMsBuild151()
```

Comment:
Yes, makes sense. This should be separate work though as there may be some iteration and lag.